test/drenv/envfile_test.py

```diff
@@ -58,9 +58,25 @@
 """
 
 
-def test_valid():
+def create_addons(tmpdir):
+    tmpdir.mkdir("addon1")
+    tmpdir.mkdir("addon2")
+    tmpdir.mkdir("addon3")
+    tmpdir.mkdir("addon4")
+    tmpdir.mkdir("addon5")
+    tmpdir.mkdir("addon6")
```

Comment:
This works, but since we use numbered addons, maybe use a loop?

     for i in range(1, 7):
         tmpdir.mkdir(f"addon{i}")